test/node/replumb/require_node_test.cljs

```diff
@@ -446,7 +561,15 @@ trim-newline
     (process-require)
     (process-goog-import)
     (ns-macro)
-    (ns-macro-with-require-macro)
+
+    (ns-macro-require-refer-macros)
+    (ns-macro-require-include-macros)
+    (ns-macro-require-macros)
+    (ns-macro-require-macros-refer)
+    (ns-macro-use-macros)
+    (ns-macro-require)
+    (ns-macro-require-macros-as)
```

Comment:
Too bad that negating tests is not that easy I wish I could do:

```
(defn test-ns-hook []
  
  (ns-macro-require-macros)
  (ns-macro-require-include-macros)
  (not (ns-macro-require-macros-refer)) 
```

Other order, but all the tests become `is-not`...